documentation/diff-reports-previous-version/GeospatialOntology.md

```diff
@@ -0,0 +1,120 @@
+
+- [Geospatial Ellipse](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialEllipse) SubClassOf [Geospatial Line String](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLineString) 
+
+#### Added
+- [Geospatial Ellipse](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialEllipse) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A One-Dimensional Geospatial Boundary that is formed by following a symmetric arc between four vertices which are connected to the center point of the ellipse via a straight line."@en 
+
+- [Geospatial Ellipse](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialEllipse) SubClassOf [One-Dimensional Geospatial Boundary](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialBoundary) 
+
+
+### Geospatial Error Region `http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialErrorRegion`
+#### Removed
+- [Geospatial Error Region](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialErrorRegion) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A Geospatial Boundary that bounds some Geospatial Region according to probability estimations for locating some object within it."@en 
+
+#### Added
+- [Geospatial Error Region](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialErrorRegion) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A One-Dimensional Geospatial Boundary that bounds some Geospatial Region according to probability estimations for locating some object within it."@en 
+
+
+### Geospatial Line String `http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLineString`
+#### Removed
+- [Geospatial Line String](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLineString) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A Geospatial Boundary that has two or more Geospatial Positions as vertices, where each vertex is connected to only one other vertex by a straight line."@en 
+
+#### Added
+- [Geospatial Line String](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLineString) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A One-Dimensional Geospatial Boundary that has two or more Geospatial Positions as vertices, where each vertex is connected to only one other vertex by a straight line."@en 
+
+
+### Geospatial Location `http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLocation`
+#### Removed
+- [Geospatial Location](http://www.ontologyrepository.com/CommonCoreOntologies/GeospatialLocation) [definition](http://www.ontologyrepository.com/CommonCoreOntologies/definition) "A Geospatial Region at which an Entity or Event is located."@en 
+
+#### Added
```

Comment:
I can't find the PR or commit where this was updated. I'm not sure why "that is" was added. The original definition seems appropriate and the replacement seems awkward.